app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
The app should have to asks for confirmation of email. To enable that kindly, add `:confirmable` here at devise.

Reply:
on it .